src/store/user/user-context.js

```diff
@@ -10,17 +10,30 @@ export const UserContextProvider = ({ children }) => {
   const [showMemberRoleUpdateModal, setShowMemberRoleUpdateModal] =
     useState(false);
   const [selectedMember, setSelectedMember] = useState(null);
+  const [showMemberTaskUpdateModal, setshowMemberTaskUpdateModal] =
```

Comment:
"setshowMemberTaskUpdateModal" can be camelCased.

Reply:
will do that